manifests/director.pp

```diff
@@ -37,11 +38,20 @@
     }
   }
 
+  unless $service_allow_restart {
+    $reload_command = $::facts['service_provider'] ? {
+      'systemd' => "systemctl reload ${service_name}",
+      default   => "service ${service_name} relaod",
```

Comment:
I think we can simplify this whole block by removing sysv support in one PR and then just support systemd based distributions

Reply:
Well, this puts me in a dilemma. Since I still have to administer some CentOS 6 machines, I will not remove the sysV support from this module.
As far as I can see, this module is quite indifferent about the service provider cause it just uses the service resource and relies on the correct config coming from the installed Bareos package. Would be nice to keep it that way.

The selector here could be easily extended for other service providers.
Alternatively the reload command could be specified via parameter, so that the service provider specific stuff can kept out of the module.

Reply:
@bastelfreak any suggestions here? I would leave it how it is

Reply:
Doesn't the provider know how to handle this?

Reply:
Well I have no idea how to handle this anyway else. The resource type `service` seems to have no understanding of reloading a service at all. https://puppet.com/docs/puppet/6.21/types/service.html
I all other modules which needs reloading (like bird) I've seen something like this.

Reply:
can you please:
* mark the PR as ready for review
* accept my suggest change to remove the `::`

I think it's safe to merge this afterwards.

Reply:
done